clap-v3-utils/src/keypair.rs

```diff
@@ -1104,29 +1106,27 @@ pub fn ae_key_from_path(
     path: &str,
     key_name: &str,
 ) -> Result<AeKey, Box<dyn error::Error>> {
-    encodable_key_from_path(matches, path, key_name)
+    let skip_validation = matches.is_present(SKIP_SEED_PHRASE_VALIDATION_ARG.name);
+    encodable_key_from_path(path, key_name, skip_validation)
 }
 
 fn encodable_key_from_path<K: EncodableKey + SeedDerivable>(
-    matches: &ArgMatches,
     path: &str,
     keypair_name: &str,
+    skip_validation: bool,
 ) -> Result<K, Box<dyn error::Error>> {
     let SignerSource {
         kind,
         derivation_path,
         legacy,
     } = parse_signer_source(path)?;
     match kind {
-        SignerSourceKind::Prompt => {
-            let skip_validation = matches.is_present(SKIP_SEED_PHRASE_VALIDATION_ARG.name);
```

Comment:
so there's some bug with `const ref`s in generic contexts that triggers ICE in the targeted toolchain. took a day to isolate and stumble into this "fix" via introducing replicode.

didn't see any related issues on github and haven't had time to try to minimize repro yet

Reply:
Have you reported this already? Even without minimal reproduction it might be worth giving them a heads up on this.

Reply:
not yet. if we can put this commit in, i'll have something to permalink. don't think the pr links persist after i delete the branch. otherwise hoping to have time to attempt a minimization tonight.